docs/setup/clouds/index.md

```diff
@@ -31,7 +31,34 @@ The currently supported cloud providers are:
 ## Supporting a new cloud provider
 
 To develop a new cloud provider add a new folder in
-`srcs/runners_manager/vm_creation` containing a `CloudManager`
-object matching [CloudManager] interface.
+`srcs/runners_manager/vm_creation` containing a
+`CloudManager` object matching [CloudManager] interface.
+
+### Retrieving VM parameters
+
+To communicate with the application, the user will provide
+a lot of information in a `yml` configuration file, such as
+`image_id`, `instance_type`, `security_group_ids`, and so on.
+During development, a `config_example.yml` file will be created
+to test the setup of the new Cloud Provider and to provide
+information about the VM. Next, to send this information to
+the new Cloud Provider, the `AwsConfigVmType` class will be
```

Comment:
As we are not specifically into the aws section, you can mention `AwsConfigVmType` but use it as an example. Or mention the base class.